Dungeoneer/src/com/interrupt/dungeoneer/GameApplication.java

```diff
@@ -90,7 +90,7 @@ public static void ShowGameOverScreen(boolean escaped) {
 				warp.generated = endingLevel.generated;
 				warp.levelToLoad = endingLevel.levelFileName;
 				warp.levelTheme = endingLevel.theme;
-				warp.fogColor = endingLevel.fogColor;
+				warp.fogColor.set(endingLevel.fogColor);
```

Comment:
`set` is used here because of GC?

Reply:
Otherwise we run the risk of changing a color that we don't intend. I often stumble with something like
```java
Color c = Color.WHITE;
...
Color d = c;
...
d.set(1f, 0f, 0f);
```
Which will set the white color to red and break all sorts of things. This is practice I want us to get into. Use new if we want a unique object. Use set if the intention is to change the color.